GitUI/Translation/English.xlf

```diff
@@ -8448,6 +8448,10 @@ See the changes in the commit form.</source>
         <source>Show superproject tags</source>
         <target />
       </trans-unit>
+      <trans-unit id="SwitchBetweenArtificialCommits.Text">
+        <source>Go to / switch between artificial commits and HEAD</source>
```

Comment:
```suggestion
        <source>Toggle between the artificial and the HEAD commits.</source>
```

Reply:
Omitted the articles because it is a menu item.